wiki/fr/fr-Command-Line-Options.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: wiki
+title: "Options en ligne de commande"
+lang: "fr"
+permalink: "/wiki/Command-Line-Options"
+---
+
+# Options en ligne de commande
+
+Si les fonctions les plus courantes de Jamulus peuvent être configurées à l'aide de l'interface graphique, Jamulus peut également être contrôlé à l'aide de ces options lorsqu'il est démarré en ligne de commande. La manière exacte de procéder dépendra de votre système d'exploitation.
+
+Par exemple, sous Windows pour activer l'enregistrement avec l'option `-R` : cliquez-droit sur le raccourci Jamulus et choisissez `Propriétés` > `Cible`. Ajoutez les arguments nécessaires au fichier `Jamulus.exe` :
+
+```shell
+"C:\Program Files (x86)\Jamulus\Jamulus.exe" -s -R "C:\chemin\vers\vos\enregistrements de bœufs"
+```
+
+Pour Macintosh, lancez une fenêtre de terminal et exécutez Jamulus avec les options souhaitées comme ceci:
```

Comment:
I know that Macintosh is used in the EN version, but it is really a wrong formulation.
But we can wait EN update.